stories/core/LabeledInput.stories.tsx

```diff
@@ -173,3 +173,32 @@ HybridLayout.args = {
   svgIcon: <SvgPlaceholder />,
   message: 'Block layout with inline icon',
 };
+
+export const HybridLayoutWithButton: Story<LabeledInputProps> = (args) => {
+  return (
+    <>
+      <LabeledInput
+        placeholder='Enter text here...'
+        label='This is a label'
+        svgIcon={
+          <IconButton styleType='borderless'>
+            <SvgCloseSmall />
+          </IconButton>
+        }
+        message='Block layout with inline borderless button'
+        iconDisplayStyle='inline'
+        {...args}
+      />
+    </>
+  );
+};
+
+HybridLayoutWithButton.args = {
+  svgIcon: (
```

Comment:
Is it needed here as `svgIcon` control is disabled?

Reply:
yeah, can remove, I guess